### ISSUE TITLE
indices from the alex-mp-20 split

### ISSUE THREAD

**Issue by bkmi** [FERMat-ML/OMatG#1]
Dear OMatG,

Thank you for the repository!

I'm interested in training with the same split as you used in Alex-MP-20 for the CSP task. In the work you said you take 10% of training data and make it into a new test set.

Since mattergen's data is available through `git lfs` https://github.com/microsoft/mattergen/tree/main/data-release/alex-mp, it would be enough to know the indices in their `train.csv` file that were converted into a test set. Would you mind releasing those please?

Ben

**Comment by PhilippHoellmer** [comment]
Hi Ben,

Thank you for your interest in OMatG.

I attached the list of 67521 material ids appearing in the MatterGen's `train.csv` file that we used to create the test set. 

Let me know if there are any further questions.

Philipp

[test_ids.txt](https://github.com/user-attachments/files/20053240/test_ids.txt)
